src/components/shop/shop.js

```diff
@@ -21,29 +23,40 @@ export class Shop extends Screen {
 
     create() {
         this.addBackgroundItems();
-        this.setLayout(["home", "pause"]);
+        this.setLayout(["back", "pause"]);
+
+        this.backMessage = this.memoizeBackButton();
+
+        this.customMessage = {
+            channel: this.backMessage.channel,
+            name: this.backMessage.name,
+            callback: () => this.setVisible("top"),
+        };
+
         const metrics = getMetrics();
-        this.title = this.createTitle(metrics);
+        this.title = createTitle(this, metrics);
         this.balance = createBalance(this, metrics);
-        this.panel = new ScrollableList(this).panel;
-        this.setupEvents();
-    }
 
-    createTitle(metrics) {
-        const titleContainer = this.add.container();
-        titleContainer.add(createTitles(this));
+        this.menus = {
```

Comment:
dunno if we should call all of these things a menu, one of them seems to be a menu?

Reply:
Changed to `panes` in 9a8904e.